library/Fission/Internal/Fixture/User.hs

```diff
@@ -22,8 +22,9 @@ user = User
 
   --
 
-  , userPublicKey = Just Ed25519.pk
-  , userDataRoot  = CID "QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ"
+  , userPublicKey    = Just Ed25519.pk
+  , userExchangeKeys = Just [Ed25519.pk]
```

Comment:
I thought we said that exchange keys need to be RSA2048 for compatability, no?

Reply:
hmmm yes good point :+1: 

Either that or we need to deterministically derive RSA2048 from Curve25519 keys. But I think just requiring RSA makes more sense

Reply:
In either case, the PK needs to be RSA. More flexible to not derive the key in case we need to rotate it? Unclear, but probably the easiest path.